sdk/storage/azblob/zt_access_policy_test.go

```diff
@@ -417,28 +322,29 @@ func (s *azblobTestSuite) TestContainerSetPermissionsACLMoreThanFive() {
 		},
 	}
 	_, err = containerClient.SetAccessPolicy(ctx, &setAccessPolicyOptions)
-	_assert.NotNil(err)
+	require.Error(t, err)
 
 	validateStorageError(_assert, err, StorageErrorCodeInvalidXMLDocument)
 }
 
-func (s *azblobTestSuite) TestContainerSetPermissionsDeleteAndModifyACL() {
```

Comment:
We have two test suites - `azblobUnrecordedTestSuite`  and `azblobTestSuite`.  We run the former suite live and later suite as recorded tests. How do these changes distinguish between them?

Reply:
I'll need your help setting up Live tests in CI too.

Reply:
Live tests will already be set up, but I can help verify they are working as expected. I included an ARM template which the CI picks up to deploy resources. In the second PR we can verify that everything is working as expected.

Reply:
> We have two test suites - `azblobUnrecordedTestSuite` and `azblobTestSuite`. We run the former suite live and later suite as recorded tests. How do these changes distinguish between them?

What is the semantic difference between the two? I didn't find failures running the unrecorded tests in playback. I can mark those tests as live only but curious what the use-case is for that. I think it's best to run all playback tests whenever possible.